server/data/mappers/referencedAndAuditableMapper.ts

```diff
@@ -0,0 +1,26 @@
+import { parseISO } from 'date-fns'
+import type { ReferencedAndAuditable } from 'dto'
+
+const toReferenceAndAuditable = (apiResponse: {
```

Comment:
Straightforward field-to-field mapping. The only complexity are the 2 date fields, as they need to be mapped from ISO string values (from the API responses) to real Date instances